libs/blocks/aside/aside.js

```diff
@@ -182,7 +188,20 @@ function decorateLayout(el) {
   }
   const picture = text?.querySelector('p picture');
   const iconArea = picture ? (picture.closest('p') || createTag('p', null, picture)) : null;
-  iconArea?.classList.add('icon-area');
+  if (iconArea) {
+    let iconClass = 'icon-area';
+    let needsIconography = false;
+    if (el.className.includes('-avatar')) {
+      iconClass = 'avatar-area';
+      needsIconography = true;
+    }
+    if (el.className.includes('-lockup')) {
+      iconClass = 'lockup-area';
+      needsIconography = true;
+    }
+    if (needsIconography) await loadIconography();
+    iconArea.classList.add(iconClass);
+  }
```

Comment:
This feels like it could be simplified... maybe something like this?

```js
  if (iconArea) {
    const iconVariant = el.className.match(/-(avatar|lockup)/);
    const iconClass = iconVariant ? `${iconVariant[1]}-area` : 'icon-area';
    if (iconVariant) await loadIconography();
    iconArea.classList.add(iconClass);
  }
```